### ISSUE TITLE
Make php support a plugin

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#7775]
Pretty sure PHP is totally broken in 3.x

Couple of things

The new CMSFilter does not call the phpservlet if the file is a php file - it forwards on to the SpeedyAsset servlet which no workee.  This was obviously caused by the new CMSfilter, which is a simple fix (though we should be thoughtful here).  

Second and more pernicious, I think the repackaging killed all the php function classes.  I am getting this error if I try to call phpinfo or date or any php function, even using the viewtool - 
`$php.eval("phpinfo();")`

```
com.dotcms.repackage.com.caucho.quercus.QuercusErrorException: /test.php:3: Fatal Error: 'phpinfo' is an unknown function.
```

Bottom line, I think we need to get php out of the core of dotCMS and make it a plugin.

**Comment by maurizo** [comment]
A new plugin has been created to handle PHP script on https://github.com/dotCMS/plugin-php

**Comment by maurizo** [comment]
Pull Request #7962

**Comment by bryanboza** [comment]
Fixed, tested on 3.3 // Postgres // FF
